homeassistant/components/device_automation/__init__.py

```diff
@@ -68,9 +71,25 @@
     hass.components.websocket_api.async_register_command(
         websocket_device_automation_get_trigger_capabilities
     )
+
+    component = hass.data[DOMAIN] = EntityComponent(
```

Comment:
We shouldn't add things to device automation to support MQTT, that seems weird. MQTT needs to organize their own storage?

Reply:
Right, `device_automation` is now handled as a special case in MQTT discovery instead.